source/puppet/5.3/hiera_config_yaml_5.md

```diff
@@ -47,6 +47,25 @@ Related topics: [$confdir][confdir], [`<MODULE>`][module].
 -   `default_hierarchy` - An array of hashes, which sets a default hierarchy to be used only if the normal hierarchy entries do not result in a value. Only allowed in a module's hiera.yaml.
 -   `defaults` - A hash, which can set a default datadir, backend, and options for hierarchy levels.
 
+Example of an environment hiera.yaml file:
+
+```
+---
+version: 5
+
+defaults:
+  datadir: "data"
```

Comment:
this is the default datadir location -no need to explicitly set the default as the default.
If here to show that you can set it, it should be commented out.